examples/tfengine/folder_foundation.hcl

```diff
@@ -28,44 +29,59 @@ data = {
 
 template "devops" {
   recipe_path = "{{$recipes}}/devops.hcl"
+  output_path = "./bootstrap"
   data = {
     # TODO(user): Uncomment and re-run the engine after generated bootstrap module has been deployed.
     # Run `terraform init` in the bootstrap module to backup its state to GCS.
     # enable_bootstrap_gcs_backend = true
 
     admins_group = "example-folder-admins@example.com"
-    state_bucket = "example-terraform-state"
 
     project = {
       project_id = "example-devops"
       owners = [
         "group:example-devops-owners@example.com",
       ]
     }
-    cicd = {
-      github = {
-        owner = "GoogleCloudPlatform"
-        name  = "example"
-      }
-      branch_regex   = "^master$"
-      terraform_root = "terraform"
+  }
+}
 
-      # Prepare and enable default triggers.
-      triggers = {
-        validate = {}
-        plan     = {}
-        apply    = {}
-      }
-      build_viewers = [
-        "group:example-cicd-viewers@example.com",
-      ]
+template "cicd" {
+  recipe_path = "{{$recipes}}/cicd.hcl"
+  output_path = "./cicd"
+  data = {
+    project_id = "example-devops"
+    github = {
+      owner = "GoogleCloudPlatform"
+      name  = "example"
+    }
+    branch_regex   = "^master$"
+    terraform_root = "terraform"
+
+    # Prepare and enable default triggers.
+    triggers = {
+      validate = {}
+      plan     = {}
+      apply    = { run_on_push = false } # Do not auto run on push to branch
```

Comment:
you can leave `run_on_push` set to default. I used `devops` example to test and demo the disabled case. But in general, `apply` should also be enabled by default.